tests/Collection/CollectionFunctionalTest.php

```diff
@@ -447,16 +448,28 @@ public function testMapReduce(): void
     public function collectionMethodClosures()
     {
         return [
-            [
+            'read-only aggregate' => [
                 function ($collection, $session, $options = []): void {
                     $collection->aggregate(
                         [['$match' => ['_id' => ['$lt' => 3]]]],
                         ['session' => $session] + $options
                     );
+                }, 'r',
```

Comment:
This pipeline is actually read-only. I added a write pipeline as @jmikola recommended.